src/firefox/index.js

```diff
@@ -208,16 +210,22 @@ export function configureProfile(
   {
     app = 'firefox',
     getPrefs = defaultPrefGetter,
-  }: ConfigureProfileOptions = {}
+    customPrefs = {},
+  }: ConfigureProfileOptions = {},
 ): Promise<FirefoxProfile> {
   // Set default preferences. Some of these are required for the add-on to
   // operate, such as disabling signatures.
-  // TODO: support custom preferences.
-  // https://github.com/mozilla/web-ext/issues/88
   let prefs = getPrefs(app);
   Object.keys(prefs).forEach((pref) => {
     profile.setPreference(pref, prefs[pref]);
   });
+  if (Object.keys(customPrefs).length > 0) {
+    log.info(`Setting custom Firefox preferences:
+      ${JSON.stringify(customPrefs, null, ' ')}`);
```

Comment:
The JSON.stringify "pretty print" mode is usually achieved with: `JSON.stringify(customPrefs, null, 2)`

Also, how about reformatting the above two lines into:
```
const customPrefsStr = JSON.stringify(...);
log.info(`Setting custom Firefox preferences: ${customPrefsStr}`);
```